doc/examples/features_detection/plot_sift.py

```diff
@@ -0,0 +1,68 @@
+"""
+==============================================
+SIFT feature detector and descriptor extractor
+==============================================
+
+This example demonstrates the SIFT feature detection and its description
+algorithm.
+
```

Comment:
It would be nice to have some context here about what SIFT is, how it is used, and what other feature detectors exist in skimage.